packages/token-balance-cache/src/providers/terra.ts

```diff
@@ -5,11 +5,15 @@ import { BalanceType } from 'common-common/src/types';
 import type { IChainNode } from '../types';
 import { BalanceProvider } from '../types';
 
-export default class TerraBalanceProvider extends BalanceProvider {
+export default class TerraBalanceProvider extends BalanceProvider<string> {
```

Comment:
What we should do here is return a function that takes an address and returns its balance (by making the query). Remember that our goal is to unit test these `getBalance` calls, so that will let us insert a mock function.